### DISCUSSION TITLE
Setting Station-MaxCurrent configuration key on Alfen charger

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [lbbrhzn/ocpp#850] [General]
Hi!

Is it possible with this integration to set the "Station-MaxCurrent" config key on the charger? If I try to change `number.charger_maximum_current` it doesn't do anything on the charger.

Im trying to dynamicaly change the current offered to the EV based on solar input, but I can't seem to figure out how/what configuration key I can use out of the box. The documentation (attached) states I can change the Station-MaxCurrent key but I don't know how.

Any help would greatly be appreciated!
[Alfen Backoffice Configuration keys 5.0.0.pdf](https://github.com/lbbrhzn/ocpp/files/11581447/Backoffice.Configuration.keys.5.0.0.1.pdf)

**Comment by drc38** [comment]
You can try using the `ocpp.configure` service to set the value of the key

**Comment by g3r0nim00** [comment]
Late response, but I was having the same issue... 
It was solved by selecting "Force Smart Charging feature profile" during the setup of the OCPP integration...
Without selecting this feature, I was no longer able to alter the charger maximum current value.

<img width="572" alt="image" src="https://github.com/user-attachments/assets/6cba9f73-c85b-4070-bc1b-f9e8900a84b5">

regards
